### PR TITLE
Added layer with Apex library.

### PR DESCRIPTION
**Pull request by atselousov**
We support nodes with V100 but don't provide tools to train models with mixed precision.

### CODE REVIEW

**Comment by atselousov** [comment]
@ayushkovskiy, there are sizes of images:
Screenshot from 2020-02-12 15-39-52

File: targets/python36-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile [hunk 1]
```diff
@@ -200,6 +200,16 @@ RUN APT_INSTALL="apt-get install -y --no-install-recommends" && \
...
 
+# ==================================================================
+# Apex for PyTorch mixed precision training
+# ==================================================================
+
+RUN mkdir /apex && \
+    cd /apex && \
+    git clone https://github.com/NVIDIA/apex && \
+    cd apex && \
+    python -m pip install -v --no-cache-dir --global-option="--cpp_ext" --global-option="--cuda_ext" ./
```
Comment (ayushkovskiy) [inline targets/python36-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile]: 1. Let's add tests
2. let's install it to the right place

Comment (atselousov) [reply targets/python36-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile]: 1) We can use tests that are provided by py library developers (see: apex/tests/L0). 
2) What is the proper place Apex must be installed to?

Comment (ayushkovskiy) [reply targets/python36-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile]: fixed both

**Pull request merged**